frontend/public/components/operator-lifecycle-manager/index.tsx

```diff
@@ -11,7 +11,8 @@ export { SubscriptionsPage } from './subscription';
 import * as operatorLogo from '../../imgs/operator.svg';
 import { InstallModeType } from './operator-group';
 
-export const appCatalogLabel = 'alm-catalog';
+export const copiedLabelKey = 'olm.copiedFrom';
```

Comment:
From the [Operator Lifecycle Manager codebase](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/api/apis/operators/v1alpha1/clusterserviceversion.go#L12).